package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "ava",
-    "lint": "npx prettier --write **/*.{js,md} --config ./package.json"
```

Comment:
Could you explain this change to me? I would've expected `**/*.{js,md}` to match every file recursively, even in the root directory.